test-cases/longevity/longevity-alternator-200GB-48h.yaml

```diff
@@ -0,0 +1,33 @@
+test_duration: 180
+prepare_write_cmd: [
+  "bin/ycsb load dynamodb -P workloads/workloada -threads 250 -p recordcount=200200300 -p fieldcount=1 -p fieldlength=1024 -p dataintegrity=true -p hdrhistogram.summary.addstatus=true", ]
+stress_cmd: [
+  "bin/ycsb run  dynamodb -P workloads/workloada -threads 250 -p recordcount=200200300 -p fieldcount=1 -p fieldlength=1024 -p operationcount=400400600 -p dataintegrity=true -p hdrhistogram.summary.addstatus=true", ]
+
+dynamodb_primarykey_type: HASH_AND_RANGE
+
+n_loaders: 3
+instance_type_db: 'i3.4xlarge'
+n_monitor_nodes: 1
+n_db_nodes: 4
+
+nemesis_class_name: 'LimitedChaosMonkey'
+nemesis_interval: 30
+nemesis_during_prepare: false
+
+user_prefix: 'alternator-3h'
+
+alternator_port: 8080
+alternator_enforce_authorization: true
+alternator_access_key_id: 'alternator'
```

Comment:
Why aren't these the defaults and if need to be changed we'll override them...
This is how it's been till now.

Reply:
the default are not to use authorization, `alternator_enforce_authorization` is in scylla yaml
`alternator_access_key_id` is what we pass into the client.
it's following the same logic as `client_encrypt` and `server_encrypt` the defaults are false.